src/components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Link } from '../Link';
+import { Flex } from '../Flex';
+import { Txt } from '../Txt';
+
+const Circle = styled(Flex)<{ isLast: boolean; emphasized?: boolean }>`
+	border: 1px solid
+		${(props) =>
+			props.isLast
+				? props.theme.colors.quartenary.light
+				: props.theme.colors.text.light};
+	border-radius: 50%;
+	width: ${(props) => (props.emphasized ? '16px' : '8px')};
+	height: ${(props) => (props.emphasized ? '16px' : '8px')};
+	background: ${(props) =>
+		props.isLast
+			? props.theme.colors.quartenary.light
+			: props.theme.colors.text.light};
+	margin-right: 8px;
+`;
+
+export interface Crumb {
+	text: string;
+	icon?: React.ReactNode;
+	onClick?: () => void;
+}
+
+export interface BreadcrumbsProps {
+	/** List of objects of type Crumb: `{text: string; icon?: React.ReactNode; onClick?: () => void;}` */
+	crumbs: Crumb[];
+	/** If true, use a larger Breadcrumbs */
+	emphasized?: boolean;
+}
+
+const BreadcrumbContent = React.memo(
+	({ icon, text }: Pick<Crumb, 'icon' | 'text'>) => {
+		return (
+			<Flex alignItems="center" justifyContent="center" maxWidth="130px">
```

Comment:
I don't like the maxWidth hard-coded, probably the parent should control the width and we can remove this?